command/meta.go

```diff
@@ -123,20 +102,6 @@ func (m *Meta) Client() (*api.Client, error) {
 		} else if m.flagClientCert != "" || m.flagClientKey != "" {
 			return nil, fmt.Errorf("Both client cert and client key must be provided")
 		}
-
-		client := &http.Client{
-			Transport: &http.Transport{
-				Proxy: http.ProxyFromEnvironment,
```

Comment:
`cleanhttp.DefaultClient()` returns a client with a transport that has those values set. Since the rest of the code now modifies that client's TLS configuration, rather than creating a new one, it's not necessary to specify those again.